kmich

```diff
@@ -173,10 +161,8 @@ fi
 llvm_krun_args=()
 ! $debug \
     || llvm_krun_args+=(--debug)
-! echo "$backend_dir" | grep -e contract-expander -e extractor -e input-creator -e output-compare &> /dev/null \
-    || llvm_krun_args+=(-c IO '\dv{SortString{}}("on")' String kore)
 [[ ! "$backend" == "llvm" ]] \
-    || llvm_krun_args+=(-c PATH "\"$(dirname $run_file)\"" String pretty)
+    || llvm_krun_args+=(-cPATH="\"$(dirname $run_file)\"")
```

Comment:
Let's rename `llvm_krun_args` to just `extra_args`, and pass them through to every sub-command.